background-charm-service/src/worker.ts

```diff
@@ -64,17 +105,17 @@ async function initialize(
   // Initialize charm manager
   manager = new CharmManager(currentSession);
 
-  console.log(`Worker: ${did} initialized`);
+  console.log(`Initialized`);
   initialized = true;
 }
 
 // FIXME(ja) should we make sure we kill the worker?
 async function cleanup(): Promise<void> {
   if (!initialized) {
-    console.log(`Worker: Not initialized, skipping cleanup`);
+    console.log(`Not initialized, skipping cleanup`);
     return;
   }
-  console.log(`Worker: Shutting down execution environment`);
+  console.log(`Shutting down execution environment`);
```

Comment:
missing did?

Reply:
This is using the `console = { log, error }` shim above, and gets a `Worker(SPACE_DID)` annotated (unrelated to the charm console annotation) -- i'll paste an example log in the PR